scale-tests/sdk.dashboard.py

```diff
@@ -0,0 +1,266 @@
+            ResourceStat("Disk available (placeholder)", "disk_limit", "decbytes", pod_type=pod_type).to_single_stat(),
+    ])
+
+
+def scheduler_row() -> Row:
+    """
+    Construct a Grafana row containing scheduler offer statistics
+    """
+    offer_metrics = [
+        "offers_processed",
+        "offers_received",
+        "declines_long",
+        "revives",
+        "revives_throttles",
+        "declines_short",
+    ]
+
+    resolution = "1m"
+
+    offer_timers = [
+        "offers_process_p50",
+        "offers_process_p90",
+        "offers_process_p99",
+        "offers_process_max",
+    ]
+
+    return Row(
+        title="Scheduler statistics",
+        panels=[
+            Graph(
+                title="cumulative offer statistics",
+                dataSource="${DS_PROMETHEUS}",
+                span=2,
+                targets=[Target(expr=sum(service_metric(m)), legendFormat=m) for m in offer_metrics],
+                steppedLine=True,
+                yAxes=YAxes(left=YAxis(format="short", decimals=0))
+            ),
+            Graph(
+                title="offer events per second [{} rate]".format(resolution),
+                dataSource="${DS_PROMETHEUS}",
+                span=2,
+                targets=[Target(expr=sum(service_metric(m, "rate", resolution)), legendFormat=m) for m in offer_metrics],
+                steppedLine=True,
+            ),
+            Graph(
+                title="offer processing time",
+                dataSource="${DS_PROMETHEUS}",
+                span=2,
+                targets=[Target(expr=service_metric(m), legendFormat=m) for m in offer_timers],
+                yAxes=YAxes(left=YAxis(format="ns"))
+            )
+        ]
+    )
+
+
+def task_row() -> Row:
+    """
+    Construct a Grafana row containing scheduler task statistics
+    """
+    task_metrics = [
+        "task_status_task_running",
+        "task_status_task_finished",
+        "task_status_task_lost",
+        "task_status_task_failed",
+    ]
+
+    resolution = "1m"
+
+    return Row(
+        title="Task statistics",
+        panels=[
+            Graph(
+                title="cumulative task statistics",
+                dataSource="${DS_PROMETHEUS}",
+                span=2,
+                targets=[Target(expr=sum(service_metric(m)), legendFormat=m) for m in task_metrics],
+                steppedLine=True,
+                yAxes=YAxes(left=YAxis(format="short", decimals=0))
+            ),
+            Graph(
+                title="task events per second [{} rate]".format(resolution),
+                dataSource="${DS_PROMETHEUS}",
+                span=2,
+                targets=[Target(expr=sum(service_metric(m, "rate", resolution)), legendFormat=m) for m in task_metrics],
+                steppedLine=True,
+                yAxes=YAxes(left=YAxis(format="short", decimals=0))
+            ),
+        ]
+    )
+
+
+# Generate the Grafana dashboard.
```

Comment:
This comment is not very useful. It's clear from the subsequent line that you're creating a dashboard...